getting-started/index.html

```diff
@@ -80,6 +80,9 @@
                         <button class="btn-distro active" data-page="content-linux-arch">Arch Linux</button>
                         <button class="btn-distro" data-page="content-linux-opensuse">openSUSE</button>
                         <button class="btn-distro" data-page="content-linux-ubuntu">Debian, Ubuntu, Mint</button>
+			<button class="btn-distro" data-page="content-linux-alpine">Alpine</button>
```

Comment:
I couldn't actually see that in Nano. Luckly, Monodevelop pointed it out quite harshly.